### PR TITLE
Bypass launcher for apps

### PR DESCRIPTION
**Pull request by aloscha**

### CODE REVIEW

File: src/Launcher/Helper/LauncherSettings.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿using Starcounter;
+
+namespace Launcher.Helper
+{
+    [Database]
```
Comment (warpech) [inline src/Launcher/Helper/LauncherSettings.cs]: According to https://github.com/StarcounterPrefabs/Guidelines/blob/master/project_structure.md, this file should be in path `src/Launcher/Database/LauncherSettings.cs`

File: src/Launcher/wwwroot/Launcher/viewmodels/SettingsPage.html [hunk 2]
```diff
@@ -1,3 +1,13 @@
 ﻿<template>
-    <h1>App Settings</h1>
+    <template id="2" is="dom-bind">
+        <h1>Launcher Settings</h1>
+        <div>
+            <label class="control-label" for="ignore-url-pattern"><strong>Bypass Launcher for URLs:</strong></label>
+            <input type="text" class="form-control" id="ignore-url-pattern" aria-describedby="ignore-url-pattern" value="{{model.IgnoreUrlPattern$::input}}">
+        </div>
+        <br/>
+        <button type="button" class="btn btn-primary" value="{{model.Save$::click}}" onclick="++this.value;">Save Launcher settings</button>
```
Comment (warpech) [inline src/Launcher/wwwroot/Launcher/viewmodels/SettingsPage.html]: Please always use `::click`, `onmousedown=` - as presented in KitchenSink.

File: src/Launcher/Helper/LauncherHelper.cs [hunk 3]
```diff
@@ -104,21 +104,23 @@ public static class LauncherHelper {
                 launcher.uri = req.Uri;
                 return launcher;
             });
-
             Handle.GET("/launcher/settings", (Request req) => {
 
                 LauncherPage launcher = Self.GET<LauncherPage>("/launcher");
+                var settings = SettingsHelper.GetSettings ?? new LauncherSettings();
+                StarcounterEnvironment.RunWithinApplication("LuncherSettingsLayout", () =>
```
Comment (warpech) [inline src/Launcher/Helper/LauncherHelper.cs]: Typo. Should be `LauncherSettings` (`Layout` seems unnecessary here)

File: src/Launcher/Helper/LauncherHelper.cs [hunk 4]
```diff
@@ -107,7 +107,7 @@ public static class LauncherHelper {
             Handle.GET("/launcher/settings", (Request req) => {
 
                 LauncherPage launcher = Self.GET<LauncherPage>("/launcher");
-                var settings = SettingsHelper.GetSettings ?? new LauncherSettings();
+                var settings = SettingsHelper.GetSettings();
                 StarcounterEnvironment.RunWithinApplication("LuncherSettingsLayout", () =>
```
Comment (warpech) [inline src/Launcher/Helper/LauncherHelper.cs]: Typo. Should be `LauncherSettings` (`Layout` seems unnecessary here)

File: src/Launcher/wwwroot/Launcher/viewmodels/SettingsPage.html [hunk 5]
```diff
@@ -1,3 +1,13 @@
 ﻿<template>
-    <h1>App Settings</h1>
+    <template id="2" is="dom-bind">
```
Comment (warpech) [inline src/Launcher/wwwroot/Launcher/viewmodels/SettingsPage.html]: `id="2"` looks redundant

**Comment by warpech** [comment]
I have added just a few comments. Please apply a fix and assign back to me.

**Comment by aloscha** [comment]
@warpech Please see the result of new commit.

File: src/Launcher/Helper/LauncherSettings.cs (hunk 1 above)
Comment (aloscha) [inline src/Launcher/Helper/LauncherSettings.cs]: Fixed

File: src/Launcher/wwwroot/Launcher/viewmodels/SettingsPage.html (hunk 5 above)
Comment (aloscha) [inline src/Launcher/wwwroot/Launcher/viewmodels/SettingsPage.html]: Fixed

**Comment by aloscha** [comment]
@warpech I change code based on your comments. Please review it again.

**Comment by warpech** [comment]
Looking good!

**Pull request merged**